examples/dataset/classification/tests/test_multiclass_classification.py

```diff
@@ -39,12 +39,14 @@ def dataset_name() -> str:
     return f"{TEST_PREFIX} - {DATASET}"
```

Comment:
Not sure the best way to do this, but change scope to `package` for both `dataset_name` fixture looks to fix the issue, explained below.

Certainly learned something today. Looks like this fixture is called twice, once for `test__upload_dataset` and again for `test__upload_results` even though they are in the same module.

The tests execution order for this `tests` directory currently is

```
test_binary_classification.py::test__upload_dataset
test_multiclass_classification.py::test__upload_dataset
test_binary_classification.py::test__upload_results
test_multiclass_classification.py::test__upload_results
```

My guess is the because of the interleaving, the modules are "imported"/"executed" twice, thus defying intuition of `module` scope.

Quote from pytest tests [docs](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#fixture-scopes)

> Note
> Pytest only caches one instance of a fixture at a time, which means that when using a parametrized fixture, pytest may invoke a fixture more than once in the given scope.